src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php

```diff
@@ -89,7 +89,12 @@ public function execute()
         }
 
         if (false === $this->search) {
-            throw new LdapException(sprintf('Could not complete search with dn "%s", query "%s" and filters "%s".', $this->dn, $this->query, implode(',', $this->options['filter'])));
+            $ldapError = '';
+            if (0 !== ($errno = ldap_errno($con))) {
```

Comment:
I'd suggest to write this as `if ($errno = ldap_errno($con)) {`

Reply:
I hoped to find something like `LDAP_ERROR_NONE` for better readability, but failed. Initial idea was

```
if (LDAP_ERROR_NONE !== ($errno = ldap_errno($con))) {
```

Without that your suggestion looks better, indeed, I'll fix it